pkg/daemon/runner_service.go

```diff
@@ -94,7 +97,8 @@ func (rom *runningOperatorManager) Run(op *core.Operator) *runningOperator {
 				break
 			}
 			i := p.Pull()
-			po := portOutput{runningOp.Handle, p, i, core.IsBOS(i), core.IsEOS(i)}
+
+			po := portOutput{runningOp.Handle, p.String(), i, core.IsEOS(i), core.IsBOS(i), p}
```

Comment:
@tidify do think it makes sense to also include the `Port.Type()` - this would allow the frontend to have `typed` data.